projects/women-in-tech/women-in-tech.data.js

```diff
@@ -0,0 +1,153 @@
+export default {
```

Comment:
Ideally we would generate the paths based on values, but for now this is great.

Reply:
I think I understand now, you mean for instance an array of values? for each year, right?